### ISSUE TITLE
AutocompletePrediction does not have structured_formatting

### ISSUE THREAD

**Issue by dimashi** [googlemaps/google-maps-services-java#244]
This is part of json response that is not copied to AutocompletePrediction:
"structured_formatting": {
        "main_text": "1 Main Street",
        "main_text_matched_substrings": [
          {
            "length": 7,
            "offset": 0
          }
        ],
        "secondary_text": "City, MA, United States"
      }
I was not able to find documentation on what this object represents, but it "looks like" first part is street address, and second is city, state, country, and I need exactly that.

**Comment by domesticmouse** [comment]
Can you please supply a reproduction case that generates this `structured_formatting`?

**Comment by dimashi** [comment]
I believe any autocomplete request returns it. For example this one:
https://maps.googleapis.com/maps/api/place/autocomplete/json?key=???????&input=1 Main Street
```"predictions": [
    {
      "description": "1 Main Street, Acton, MA, United States",
      "id": "3592e3b0cb5087b46f15781773166a8725238b26",
      "matched_substrings": [
        {
          "length": 1,
          "offset": 0
        },
        {
          "length": 11,
          "offset": 2
        }
      ],
      "place_id": "ChIJNZASYb2R44kRVaiUqQsdqoQ",
      "reference": "CkQ_AAAAh7dqgo2Yh9nk_A_B-_PWkkk0v_xeMkVbDowUFKs3s3nAOuwkFLrgqaWD6wDcVI9K2Du4IG6aPk6yqCExBNJZXRIQ1fVBJMRlqR1x83GhWy72JhoUZrXhCWpPDp2hwgQocglNNuxQcUc",
      "structured_formatting": {
        "main_text": "1 Main Street",
        "main_text_matched_substrings": [
          {
            "length": 1,
            "offset": 0
          },
          {
            "length": 11,
            "offset": 2
          }
        ],
        "secondary_text": "Acton, MA, United States"
      },```

**Comment by domesticmouse** [comment]
This will be fixed in v0.2.2
